products/knownprojects/sql/combine.sql

```diff
@@ -414,30 +414,220 @@ FROM (
         NULL::numeric AS inactive,
         NULL::text AS no_classa
     FROM (
-        SELECT * FROM _dcp_application
+        SELECT
+            source,
```

Comment:
Save us, dbt!

Reply:
cannot overstate how much this super union tripped me up (thinking I was fixing all geometries in the final table when it was only one of the tables pre-union)